### ISSUE TITLE
Entities for operating time and on/off cycles

### ISSUE THREAD

**Issue by 1stDeadeye** [signalkraft/mypyllant-component#87]
### Before submitting a new issue

- [X] I'm not using the older Vaillant sensoAPP (which is unsupported by this integration)
- [X] I [redownloaded the latest release](https://my.home-assistant.io/redirect/hacs_repository/?owner=signalkraft&repository=mypyllant-component) and restarted Home Assistant
- [X] I've [enabled debug logs](https://signalkraft.com/mypyllant-component/3-contributing/#debugging) and attached the relevant logs to this ticket

### Problem description

In my myVaillant app, the operating time and on/off cycles are displayed under System info. Is it possible to provide these values as an entity? This would make it easy to keep an eye on the burner starts per hour/day.

Setup: ecoTEC plus VC 15CS + sensoDirect 710 + sensoNET VR921

![screen-vc](https://github.com/signalkraft/mypyllant-component/assets/6626117/628f9ec4-8568-48d7-9fe5-fa108dbd7a51)

**Comment by signalkraft** [comment]
Could you [generate test data](https://signalkraft.com/mypyllant-component/3-contributing/#contributing-test-data) and attach it? I don't have this data in my setup.

**Comment by signalkraft** [comment]
Thanks for the data, I gave it a shot here: https://github.com/signalkraft/mypyllant-component/releases/tag/v0.7.2b2

You need to enable beta release in HACS to install this version. I don't know how deviceId relates to the rest of the data. There's a similar data field called deviceUuid, so I used that. Let me know if it works, you should see two new sensor for on/off cycles and operation time. If those don't appear, you can search for "statistics" in Developer tools > States > Filter attributes.

**Comment by signalkraft** [comment]
Should be fixed in the latest pre-release!

**Comment by kdupke** [comment]
EDIT: Solved. Ignore it. I found the cycles as `sensor.vaillant_warmepumpe_device_1_arotherm_plus_on_off_cycles`


Sorry to comment on this old issue.
I am running 0.9.19 (latest I think), Vaillant aroTHERM plus, with VR940 Internet Gateway.
I do not find any data on cycles, while the app does show it.
Was this dropped?
Or what entity is it?
I do have a total of 6 devices with 55 entities. Searching for cycle does not bring up any suitable entity.

Is there any data I need to provide? Or shall I open a fresh issue?

Best regards,
kai
